beeline-examples/package.yaml

```diff
@@ -44,6 +44,8 @@ when:
         - -Wno-all-missed-specialisations
         - -Wno-missed-specialisations
         - -Wno-unticked-promoted-constructors
+        - -Wno-missing-role-annotations
+        - -Wno-term-variable-capture
```

Comment:
These will need to be gated by the ghc version. Which is why the build failed.

Reply:
I have added the gating!